README.md

```diff
@@ -260,8 +260,8 @@ Once the package is installed and similar to what you would do for a standard La
 ```php
 'connections' => [
     // ...
-    'sqs-sns' => [
-        'driver' => 'sqs-sns',
+    'sqs_pub_sub' => [
+        'driver' => 'sqs_pub_sub',
         'key' => env('AWS_ACCESS_KEY_ID'),
         'secret' => env('AWS_SECRET_ACCESS_KEY'),
         'prefix' => env('SQS_SNS_PREFIX', 'https://sqs.us-east-1.amazonaws.com/your-account-id'),
```

Comment:
Surely there is more to update in the `README.md`, no? Basically the PR description could need to go in there. 

Which makes me wonder if we should either:
- Move the docs to the wiki on the repo
- Split pub and sub in two different repos